content/articles/mongo-nodejs-cubejs-dashboard/index.md

```diff
@@ -0,0 +1,249 @@
+### Introduction
+
+In this tutorial, I'll show you how to create analytics dashboard. We'll be using Node.js, Express, and Cube.js, an open-source tool for creating API analytics.  We will also use MongoDB as our database to store and retrieve data together with the MongoDB BI Connector.
+
+### Table of contents
+
+- [Introduction](#introduction)
+- [Table of contents](#table-of-contents)
+- [Prerequisites](#prerequisites)
+- [Objectives](#objectives)
+- [Setting up the development environment](#setting-up-the-development-environment)
+- [Generating Cube.js schema](#generating-cubejs-schema)
+- [Adding Cube.js environmental variables](#adding-cubejs-environmental-variables)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+
+- This tutorial requires a basic understanding of the MongoDB and MongoDB BI connector.
+- Basic knowledge in Node.js, Express, and Cube.js.
```

Comment:
@bhanjibrilliant, could give links to cube.js. Or rather explain what it does. Do not let readers scratch their heads over something you could as well explain. I found a few instances of this, kindly fix them